parm/aero/berror/aero_diagb.yaml.j2

```diff
@@ -53,6 +60,8 @@ variables:
   - mass_fraction_of_sea_salt004_in_air
 
 rescale: 2.0    # rescales the filtered std. dev. by "rescale"
+diagb_weight: 0.5
+rescale_staticb: 2.0
```

Comment:
```suggestion
staticb rescaling factor: 2.0
```

Reply:
Same comment here. The underscore is needed for the variable. This one might be less confusing though.